interfaces/builtin/opengl.go

```diff
@@ -58,8 +58,12 @@ const openglConnectedPlugAppArmor = `
 # Main bi-arch GL libraries
 /var/lib/snapd/hostfs/{,usr/}lib{,32,64,x32}/{,@{multiarch}/}{,nvidia*/}lib{GL,GLU,GLESv1_CM,GLESv2,EGL,GLX}.so{,.*} rm,
 
+# Allow access to all cards since a) this is common on hybrid systems, b) ARM
+# devices commonly have two devices (such as on the Raspberry Pi 4, one for KMS
+# and another that does not) and c) there is nothing saying that /dev/dri/card0
+# is the default card or the application is currently using.
 /dev/dri/ r,
-/dev/dri/card0 rw,
+/dev/dri/card[0-9]* rw,
```

Comment:
is there a `/dev/dri/card`? or should this really be

```suggestion
/dev/dri/card[0-9]+ rw,
```